pkg/agent/openflow/client.go

```diff
@@ -294,6 +294,26 @@ type Client interface {
 	UninstallTrafficControlReturnPortFlow(returnOFPort uint32) error
 
 	InstallMulticastGroup(ofGroupID binding.GroupIDType, localReceivers []uint32) error
+
+	// InstallMulticlusterNodeFlows installs flows to handle cross-cluster packets between a regular
+	// Node and a local Gateway.
+	InstallMulticlusterNodeFlows(
+		cacheKey string,
```

Comment:
Is the key per Gateway Node when we have multiple Gateways later? And are we using Gateway Node name? If so, name it gatewayName.

Reply:
It's actually a string like `getCacheKeyWithPrefix(gw.Name+ciImport.Spec.ClusterID)` because here we need to distinguish both Gateway and remote Cluster ID.

Reply:
I feel we can still call it gatewayName or gatewayID.